planetiler-core/src/main/java/com/onthegomap/planetiler/util/LoopLineMerger.java

```diff
@@ -75,16 +77,48 @@ private void merge() {
       if (node.getEdges().size() == 2) {
         Edge a = node.getEdges().getFirst();
         Edge b = node.getEdges().get(1);
+        mergeTwoEdges(a, b);
         node.getEdges().clear();
```

Comment:
mergeTwoEdges should probably take care of removing the edges from the middle node

Reply:
OK